### PR TITLE
Added template

### PR DESCRIPTION
**Pull request by 500apps-RahulVangari**

### CODE REVIEW

**Comment by subramaniyamkolluri** [comment]
video

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+4. The container holds all your tags, such as the Google Analytics tag, Google Ads tag, or Botup tag. Choose the target platform to web and click on the Continue button.
+
+5. Read through the GTM Terms of Service. Click Yes to accept them and continue.
+
+6. You will get a container with a code snippet. Copy the code and paste it into every page of your site in the recommended places. Click Ok to close the 
+code snippet box and finish setting up the GTM manager and your first container.
+
+
+[Get Started](https://infinity.500apps.com/convertpath) to build interactive popups in [Convertpath](https://convertpath.com)
```
Comment (subramaniyamkolluri) [inline README.md:18]: ConvertPath

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+ 
+### Installation
+Adding the Convertpath tag to your website once you're logged in to your account with the container, it's time to add a new tag with the Convertpath tracking code inside it. To do so, follow these steps
```
Comment (subramaniyamkolluri) [inline README.md:22]: everywhere

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+1. In the Google Tag Manager Home, click on the New Tag tile.
+
+2. Click on the Tag Configuration tile and search for our community template 'Convertpath by 500apps (Smart Website Popups)' in the 'Community Template Gallery'.
+
+3. Click on ‘Add to workspace’ to select the template and add it to your workspace
+
+4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
+
+5. Find the randomly generated Convertpath key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md:32]: Just say api key

Its in better instructions not share script slide bar

Its not slide, its side

File: README.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+7. Save your tag and click on “Publish” so the tag information is pushed to the website through the GTM snippet previously pasted in your website.
+
+8. That’s it. You can now visit your website and start using Convertpath.
+
+
+### Support
+Contact our support team through “Live Chat” or “Help us improve” options through our application.
+
+
+![alt text](https://infinity.500apps.com/img/pushninja/GTM-Image-1.png)
```
Comment (subramaniyamkolluri) [inline README.md:45]: pushninja?

File: template.tpl [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+
+___INFO___
+
+{
+  "type": "TAG",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "Convertpath by 500apps",
```
Comment (subramaniyamkolluri) [inline template.tpl:16]: everywhere

**Review by subramaniyamkolluri — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (500apps-RahulVangari) [reply README.md:45]: This are same images so using them.

**Comment by 500apps-RahulVangari** [comment]
> video

video : https://drive.google.com/file/d/13QRDPXa751Y0g4lizxAApF5zXwxzCN_8/view

File: README.md (hunk 4 above)
Comment (subramaniyamkolluri) [reply README.md:45]: why then ninjacapture or botup had different images

Comment (500apps-RahulVangari) [reply README.md:45]: In botup they are also showing same images but they are using different files.

Comment (500apps-RahulVangari) [reply README.md:45]: They are showing same images but different files

File: README.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ Adding the Convertpath tag to your website once you're logged in to your account
 
 4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-5. Find the randomly generated ConvertPath key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
+5. Find the randomly generated api key for the department in the share script side bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md:32]: remove randomly generated phrase

File: README.md [hunk 7]
```diff
@@ -1 +1,54 @@
...
+
+1. In the Google Tag Manager Home, click on the New Tag tile.
+
+2. Click on the Tag Configuration tile and search for our community template 'ConvertPath by 500apps (Smart Website Popups)' in the 'Community Template Gallery'.
+
+3. Click on ‘Add to workspace’ to select the template and add it to your workspace
+
+4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
+
+5. Find the generated api key for the department in the share script side bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md:32]: User may not know where and how to copy api key. Instead ask him to clik gtm icon instructions and ask him to copy from there

**Review by subramaniyamkolluri — CHANGES_REQUESTED** [review]

**Comment by subramaniyamkolluri** [comment]
Manohar. Unable to tag you here. Pl review

**Comment by agilecrm** [comment]
500apps ConvertPath

**Comment by 500apps-RahulVangari** [comment]
> 500apps ConvertPath

Sir, Done with changes

File: README.md [hunk 8]
```diff
@@ -1 +1,54 @@
-# -500apps-public-gtm-convertpath
\ No newline at end of file
+# 500apps ConvertPath
+Get Started with a powerful platform to build a Smart Website Popups software for your website
```
Comment (agilecrm) [inline README.md:2]: Better content here

Comment (500apps-RahulVangari) [reply README.md]: Done sir

**Pull request merged**